tabbycat/utils/tables.py

```diff
@@ -726,9 +726,9 @@ def construct_venue_cell(venue):
 
             if len(descriptions) > 0:
                 if len(descriptions) == 1:
-                    categories_sentence = _("This venue %(predicate)s.") % {'predicate': descriptions[0]}
```

Comment:
Now that I'm looking at this again, I guess this data structure makes no sense in a translated string, since the predicate doesn't also change languages, lol.

Reply:
What do you mean, "data structure"? By the predicate not changing languages, the grammar of other languages might place them elsewhere.

Reply:
I mean the fact that we ask users to provide a predicate, e.g. "has step-free access", which would show up as "This venue has step-free access" in English. But "Esta sala has step-free access" doesn't make very much sense 😂

We should probably just abolish the "description" field and rely on category names only.

Reply:
(This is not related to this PR, I just noticed it while reviewing the code changes.)